backend/dataall/modules/omics/services/omics_enums.py

```diff
@@ -0,0 +1,6 @@
+import enum
```

Comment:
In more recent versions enums are part of the `api` package and use the class from dataall.base.api.constants import GraphQLEnumMapper